m4_sessions/1804inning/ethics_of_AI.md

```diff
@@ -6,12 +6,42 @@ Ellen Cornelius
 Andy Dymond
 Matt Walker
 
-# Name of Session: 
+# Name of Session: AI Carumba!
 
 ## Session Description (Why should students come to this session):
-
+  Come to discuss and share opinions re: the ever increasing presence of AI in the world and how to handle the ethics of AI in our roles as developers.
```

Comment:
Additional resource for you: https://github.com/turingschool/gear-up/blob/master/m4_sessions/1801-inning/Ethics_In_AI.md

https://github.com/turingschool/gear-up/blob/master/m4_sessions/1708-inning/The_Robots_are_coming.md